src/Plugin/Bitbucket.php

```diff
@@ -0,0 +1,318 @@
+<?php
+
+namespace PHPCensor\Plugin;
+
+use GuzzleHttp\Client;
+use PHPCensor\Builder;
+use PHPCensor\Database;
+use PHPCensor\Model\Build;
+use PHPCensor\Plugin;
+use PHPCensor\Plugin\Util\BitbucketPluginResult;
+use PHPCensor\Store\BuildErrorStore;
+use PHPCensor\Store\BuildStore;
+use PHPCensor\Store\Factory;
+
+class Bitbucket extends Plugin
+{
+    /** @var string */
+    protected $url;
+
+    /** @var string */
+    protected $token;
+
+    /** @var string */
+    protected $projectKey;
+
+    /** @var string */
+    protected $repositorySlug;
+
+    /** @var bool */
+    protected $createTaskPerFail = true;
+
+    /** @var bool */
+    protected $createTaskIfFail = true;
+
+    /** @var bool */
+    protected $updateBuild = false;
+
+    /** @var string */
+    protected $message = '';
+
+    /**
+     * @var Client
+     */
+    protected $httpClient;
+
+    /**
+     * @var Database
+     */
+    protected $pdo;
+
+    /**
+     * @return string
+     */
+    public static function pluginName()
+    {
+        return 'bitbucket';
```

Comment:
I saggest to rename plugin to `BitbucketNotify` (`bitbucket_notify`).

Reply:
i agree